nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/resources/docs/org.apache.nifi.processors.gcp.drive.ListGoogleDrive/additionalDetails.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+
+<head>
+    <meta charset="utf-8"/>
+    <title>ListGoogleDrive</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
+</head>
+<body>
+
+<h1>Accessing Google Drive from NiFi</h1>
+
+<p>
+    This processor uses Google Cloud credentials for authentication to access Google Drive.
+    The following steps are required to prepare the Google Cloud and Google Drive accounts for the processors:
+</p>
+<ol>
+    <li><b>Enable Google Drive API in Google Cloud</b> - Follow instructions at
+        https://developers.google.com/workspace/guides/enable-apis
+        and search for 'Google Drive API'.
+    </li>
+    <li><b>Grant access to Google Drive folder</b> - In Google Cloud Console navigate to IAM & Admin -> Service
```

Comment:
I think it would make sense to add bullet points for the sub-steps (even if the given step contains only one point).
```
<li><b>Step N</b>
    <ul>
        <li>Sub-step 1</li>
        <li>Sub-step 2</li>
        <li>...</li>
    </ul>
<li>
```